corehq/apps/app_manager/templates/app_manager/partials/xpathValidator.html

```diff
@@ -1,23 +1,19 @@
 {% load hq_shared_tags %}
+{% load i18n %}
 <script src="{% static 'xpath/lib/biginteger.js' %}"></script>
 <script src="{% static 'xpath/lib/schemeNumber.js' %}"></script>
 <script src="{% static 'xpath/xpath.js' %}"></script>
 <script src="{% static 'xpath/parser.js' %}"></script>
 <script src="{% static 'app_manager/js/xpathConfig.js' %}"></script>
 <script type="text/html" id="XPathValidator.template">
-    <div class="span3 js-xpath-input-target">
+    <div class="js-xpath-input-target control-group" data-bind="css: {error: xpathValidator.error}">
         <input type="text"
             data-bind="attr: {name: input.getAttribute('name'), placeholder: input.getAttribute('placeholder')},
-                       value: xpathValidator.xpathText, valueUpdate: 'textchange'"/>
-    </div>
-    <div class="span1">
-        <div data-bind="if: xpathValidator.xpathText() && !xpathValidator.error()">
-            <i class="icon icon-ok"></i>
-        </div>
-        <div data-bind="if: xpathValidator.error">
-            <i class="icon icon-exclamation-sign"
-               data-bind="popover: {title: xpathValidator.errorTitle, trigger: 'hover', html: true, content: xpathValidator.errorHtml}"></i>
+                       value: xpathValidator.xpathText"/>
+        <!--ko if: xpathValidator.error()-->
+        <div class="alert alert-error" data-bind="html: errorHtml">
```

Comment:
If you make this a `.help-block` instead of an alert, it'll be spaced better, and the `.error` on the parent `.control-group` should ensure that it's displayed in red.